DaBank/DaBank/Controllers/DepoWithController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DaBank.Models;
+using DaBank.Repository;
+using Microsoft.AspNetCore.Mvc;
+
+namespace DaBank.Controllers
+{
+    public class DepoWithController : Controller
+    {
+
+        private readonly IBankRepository _repository;
+
+        public DepoWithController(IBankRepository repository)
+        {
+            _repository = repository;
+        }
+
+        public IActionResult DepoWith(string error)
+        {
+            if (error != null)
+            {
+                ViewBag.Error = error;
+            }
+            return View();
+        }
+
+        public IActionResult Transaction(string submitButton, decimal value, int account)
+        {
+            var bankController = new BankController();
+
+            var requestedAccount = _repository.GetAccount(account);
+
+            var bank = _repository.GetBank();
+
+            if (requestedAccount != null)
```

Comment:
"Det ska synas ett felmeddelande om man skriver in ett kontonummer som inte finns"

Tror du får lägga in nåt felmeddelande i else-sats

Reply:
Glömde D: